src/models/hooks/activityReport.js

```diff
@@ -237,7 +238,6 @@ const determineObjectiveStatus = async (activityReportId, sequelize, isUnlocked)
         include: [{
           model: sequelize.models.Objective,
           as: 'objective',
-          attributes: ['id', 'status'],
```

Comment:
Do we want all back here or should it be attributes: []

Reply:
we want the whole thing, as we are modifying and saving the model directly below instead of just using the ids to run an update

Reply:
👍